tools/sync/command.py

```diff
@@ -163,7 +168,8 @@ def close(self):
                 return
 
         timeout_thread = None
-        output_thread = OutputThread()
+        sema = asyncio.Semaphore(value=0)
```

Comment:
The completion of an `OutputThread` seems more strictly Boolean than countable; can perhaps `threading.Event` be used here versus a counting `Semaphore`?

Reply:
I think so. It seems the semantics of set() vs. wait() should be the same as for semaphores - if the internal flag is set, then wait() will not block at all.